webapp/scripts/functions.js

```diff
@@ -238,6 +266,23 @@ module.exports = {
     return list
   },
 
+  /**
+   * get all files of a file type in a directory 
+   * @param {string} dir directory to search in
+   * @param {array} files array of filenames 
+   * @returns {array} array of filenames
+   */
+  getAllFile(dir, files, extension) {
```

Comment:
please rename this to better reflect the meaning of "get all files of a file type in a directory"

Reply:
`files` should not be a function argument, instead it should be initialized in the function itself